.cloudbuild/graalvm/cloudbuild-test-a-downstream-kms.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 7200s # 2 hours
+substitutions:
+  _SHARED_DEPENDENCIES_VERSION: '3.30.1-SNAPSHOT' # {x-version-update:google-cloud-shared-dependencies:current}
+  _JAVA_SHARED_CONFIG_VERSION: '1.7.7'
```

Comment:
This should be handled by renovate-bot ([config](https://github.com/googleapis/sdk-platform-java/blob/ea80d1cac8ab8456e764d8d1b2bedeaeebb757d4/renovate.json#L62-L70)) automatically but maybe we can double check when a new java-shared-config update shows up.

Reply:
Agreed